target/linux/generic/patches-4.4/308-mips32r2_tune.patch

```diff
@@ -1,17 +0,0 @@
-Add -mtune=34kc to MIPS CFLAGS when building for mips32r2
```

Comment:
I checked the issue again, and it seems that compiling with -march=mips32r2 -mtune=34kc works now as expected on my system. I made an complete rebuild (also the toolchain) ... maybe some bug in gcc in december 2016 was the reason for that issue. But removing the patch solved it 2016 for me... but now, it builds fine, even with the patch ;-)